cmd/tracee-ebpf/internal/flags/flags-cache.go

```diff
@@ -0,0 +1,67 @@
+package flags
+
+import (
+	"fmt"
+	"github.com/aquasecurity/tracee/pkg/events/queue"
+	"strconv"
+	"strings"
+)
+
+func CacheHelp() string {
+	return `Select different cache types for the event pipeline queueing.
+Possible options:
+cache-type={none,mem}                              pick the appropriate cache type.
+mem-cache-size=256                                 set memory cache size. only works for cache-type=mem.
```

Comment:
I think it's good to indicate unit of measure, but maybe we could document it in the help text instead. `mem-cache-size-bytes` is a bit of a mouthful IMO.

Reply:
Yep, I agree to Nadav here. mem-cache-size-bytes is too long. Documenting might be a better approach.

Reply:
I would prefer a longer name that provides all the needed information, instead of a shorter that requires me to read the documentation. Maybe we can make the name shorter and adding the unit of measure in it.

Reply:
Help command describes it is in MB. About having MB in the comm argument itself, I know no other unix tool that does that so I preferred to keep as is. I made it clear to be MB in the help AND changed variable name as requested (which, for the programmer, I think might be better).